### DISCUSSION TITLE
How to make a hard fork?

### DISCUSSION THREAD

**Discussion by Eduu84** [Cyfrin/foundry-full-course-cu#1983] [General]
I know my question has nothing to do with the content of the course (partly yes).  
I was thinking about making a fork of kaspa (in its new code written in rust), I've been researching its code for a few days but my knowledge is not enough to get it. I have found some parameters to modify such as the rpc ports for mainnet, testnet, etc... but I think I still have a long way to go.
Can someone help me in this new learning process?

**Comment by EngrPips** [comment]
Honestly, I don't even know what `kaspa` is. I suggest you do more research as this seems like your personal exploration. Have Fun while discovering a lot!

**Reply by Eduu84** [reply]
https://coinmarketcap.com/es/currencies/kaspa/

https://github.com/kaspanet/rusty-kaspa

:)

**Reply by EngrPips** [reply]
Thanks for the resources, Friend.
